src/server/qgsserversettings.h

```diff
@@ -183,6 +185,20 @@ class SERVER_EXPORT QgsServerSettings
      * \since QGIS 3.8
      */
     bool showGroupSeparator() const;
+  
+    /**
+     * Returns the max height of a WMS GetMap request.
```

Comment:
Maybe also add a comment here that this acts as a limit for any project configuration?

Reply:
Due to the recent work of @signedav in https://github.com/qgis/QGIS/pull/9545, it'll also have an impact on `GetLegendGraphics` request. I don't know if it's wanted, but in this case, it should be stated in the documentation too.

Reply:
@pblottiere I think it is fair to use it also on the GetLegendGraphics, since the settings in the project would also impact that. no?

Reply:
Hi @mbernasocchi,

> I think it is fair to use it also on the GetLegendGraphics, since the settings in the project would also impact that. no?

Good point :).

And thanks for your work!